websites/G/Gartic.io/presence.ts

```diff
@@ -1,38 +1,47 @@
-const presence = new Presence({
-		clientId: "808668919635247104",
-	}),
-	browsingTimestamp = Math.floor(Date.now() / 1000);
+const presence = new Presence({ clientId: "808668919635247104" }),
+	garticPlayingTimestamp = Math.floor(Date.now() / 1000),
+	getGarticPresence = (): PresenceData => {
+		const path = document.location.pathname,
+			base: PresenceData = {
+				largeImageKey: "https://i.imgur.com/3rfHXtU.png",
+				startTimestamp: garticPlayingTimestamp,
+			};
+
+		if (path === "/") return { ...base, details: "Viewing the homepage" };
+
+		if (path === "/rooms") return { ...base, details: "Browsing rooms" };
+
+		if (path !== "/room" && !path.match(/^\/[0-9a-zA-Z]{6,}/))
+			return { ...base, details: "Somewhere on the site" };
```

Comment:
maybe add a lil comment here for the regex (so future contributors understand why its there)